deploy/cluster_role.yaml

```diff
@@ -21,7 +21,10 @@ rules:
   resources:
   - endpoints
   - clusters
+  - gateways
```

Comment:
This cluster_role.yaml is only for globalnet I think. It may not need to change the permissions

Reply:
OK, I’ll remove these.